### PR TITLE
Updated Descriptions For Employees

### PR DESCRIPTION
**Pull request by Logan-Ruf**
Each current employee description has been added. Format for the description should work well on most screens.

### CODE REVIEW

**Comment by BrendanMyers88** [comment]
Screen Shot 2020-08-24 at 11 27 10 AM

Losing some text on Eric, Bailey, Logan, and Nic's profile for a MacBook Screen (1440x756)

File: about_us.html [hunk 1]
```diff
@@ -82,29 +82,60 @@
...
-                                </div>
-                            </v-row>
-                            <v-row>
-                                <div class="skills">
-                                    <ul v-for="skill in slide.skills">
-                                        <li>{{skill.name}} : {{skill.power_level}}</li>
-                                    </ul>
-                                </div>
+                            <v-row class="description-row">
+                                <v-col class="description-sizer" cols="12" md="6" style="text-align: left;">
```
Comment (BrendanMyers88) [inline about_us.html]: Can we move this `text-align: left;` out of the component and into the css?

File: about_us.html [hunk 2]
```diff
@@ -82,29 +82,60 @@
...
+                                        {{slide.name}}
+                                    </div>
+                                    <div class="desc-title">
+                                        {{slide.title}}
+                                    </div>
+                                    <div class="desc-title">
+                                        Top Skills:
+                                    </div>
+                                    <div class="skills">
+                                        <ul style="list-style-type: none">
```
Comment (BrendanMyers88) [inline about_us.html]: Pull inline styling and move it to css

```
.skills > ul {
    list-style-type: none;
}

**Comment by Logan-Ruf** [comment]
> Screen Shot 2020-08-24 at 11 27 10 AM
> 
> Losing some text on Eric, Bailey, Logan, and Nic's profile for a MacBook Screen (1440x756)

That was happening on small screen, but I didn't see it for the larger screens. Should be able to use the "description-sizer" class to make sure the column is the same size as the background.

File: about_us.html [hunk 3]
```diff
@@ -82,29 +82,60 @@
...
+                                            <li v-for="skill in slide.skills"><b>{{skill.type}}</b>{{skill.skills}}</li>
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
                                            <li v-for="skill in slide.skills"><strong>{{skill.type}}</strong>{{skill.skills}}</li>
```
`<strong>` tag is preferred to `<b>` tag now.

File: about_us.html [hunk 4]
```diff
@@ -82,29 +82,60 @@
...
+                                        </ul>
+                                    </div>
+                                </v-col>
+                                <v-col class="description-sizer" style="text-align: left" cols="12" md="6">
```
Comment (BrendanMyers88) [inline about_us.html]: Pull inline styling here

File: about_us.html [hunk 5]
```diff
@@ -82,29 +82,60 @@
...
+                                    <div class="desc-links desc-links-before">
+                                        <template v-if="slide.website != ''">
```
Comment (BrendanMyers88) [inline about_us.html]: We can change all these from `v-if="slide.website != ''"` to `v-if="slide.website"` or that equivalent. An empty string evaluates to false so we can just check if there's a string value in our `slide.[link]`

File: about_us.html [hunk 6]
```diff
@@ -174,113 +205,214 @@
                             src: "./assets/images/avatars/eric.png",
                             name: "Eric Carmichael",
                             title: "Owner, Lead Developer",
-                            description: "Lorem ipsum dolor sit amet.",
+                            description: ["Eric started programming at a young age. At the age of 15, he fell in love with it when hacked his school’s school stock exchange. CKC was founded in 2012 so Eric could have the flexibility to work and take care of his mother’s health. Since then we’ve had the opportunity to work with some of the largest Django shops and run projects from wireframes to millions in funding.",
```
Comment (BrendanMyers88) [inline about_us.html]: I'd like to move this slides array out of our HTML and into its own file that we can import so we don't have a massive text dump in the middle of our Vue file.

File: about_us.html [hunk 7]
```diff
@@ -470,23 +606,55 @@
 
     .description-bg .name {
         color: #F06C7C;
-        font-size: 24px;
-        padding: 10px 20px;
+        font-size: 36px;
+        padding: 0px 20px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
        padding: 0 20px;
```

File: about_us.html [hunk 8]
```diff
@@ -470,23 +606,55 @@
...
     .description-bg .desc-title {
         font-size: 24px;
         padding: 0 20px;
     }
 
-    .description-bg .description {
+    .description-bg .skills {
         font-size: 20px;
-        padding: 0 20px;
+        padding: 0px 20px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
        padding: 0 20px;
```

File: about_us.html [hunk 9]
```diff
@@ -512,13 +707,13 @@
 
 
         .image-holder {
-            margin-bottom: 0;
-            width: calc(31.532vh + -81.081px);
-            height: calc(31.532vh + -81.081px);
+            margin-bottom: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-bottom: 0;
```

File: about_us.html [hunk 10]
```diff
@@ -530,41 +725,72 @@
         .right-arrow {
             position: absolute;
             cursor: pointer;
-            margin-top: 0;
+            margin-top: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-top: 0;
```

File: about_us.html [hunk 11]
```diff
@@ -530,41 +725,72 @@
...
             top: calc(53.874vh + -263.676px);
         }
 
         .right-arrow {
-            margin-left: 0;
+            margin-left: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-left: 0;
```

File: about_us.html [hunk 12]
```diff
@@ -530,41 +725,72 @@
...
             right: calc(23vw - 100px);
         }
 
         .left-arrow {
-            margin-right: 0;
+            margin-right: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-right: 0;
```

File: about_us.html [hunk 13]
```diff
@@ -585,7 +811,7 @@
 
 
         .image-holder {
-            margin-bottom: 0;
+            margin-bottom: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-bottom: 0;
```

File: about_us.html [hunk 14]
```diff
@@ -603,30 +829,28 @@
         .right-arrow {
             position: absolute;
             cursor: pointer;
-            margin-top: 0;
+            margin-top: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-top: 0;
```

File: about_us.html [hunk 15]
```diff
@@ -603,30 +829,28 @@
...
             top: calc(17.333vh + 121.000px);
         }
 
         .right-arrow {
-            margin-left: 0;
+            margin-left: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-left: 0;
```

File: about_us.html [hunk 16]
```diff
@@ -603,30 +829,28 @@
...
             right: calc(23vw - 100px);
         }
 
         .left-arrow {
-            margin-right: 0;
+            margin-right: 0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            margin-right: 0;
```

File: about_us.html [hunk 17]
```diff
@@ -754,7 +1032,7 @@
             width: 100vw;
             height: 600px;
             background-size: 190px;
-            background-position-y: 0;
+            background-position-y: -0px;
```
Comment (BrendanMyers88) [inline about_us.html]:
```suggestion
            background-position-y: 0;
```

**Review by BrendanMyers88 — CHANGES_REQUESTED** [review]
Needs some fixes for inline styling, css values, and I think we should move the data array out of our Vue file and into its own file. Other than that it's looking pretty good. Needs a minor visual fix on a couple lines that are being cut off but other than that great!

File: about_us.html (hunk 6 above)
Comment (BrendanMyers88) [reply about_us.html]: I moved this array in the latest push, make sure you pull from the branch before you re-push

**Comment by Logan-Ruf** [comment]
Committed changes from review. This should be good to merge.

**Review by ckcollab — APPROVED** [review]

**Pull request merged**